contents/handbook/people/team-structure/core-analytics.md

```diff
@@ -8,6 +8,8 @@ hideAnchor: true
 ## People
 
 - [Eric Duong](/handbook/people/team#eric-duong-software-engineer) (Team Lead, Full Stack Engineer)
+- [Karl-Aksel Puulmann](/handbook/people/team#karlaksel-puulmann-software-engineer) (Full Stack Engineer)
+
```

Comment:
Also Neil! Looks like a bigger update regarding team structure is overdue

Reply:
Wait where are you? lol

I just moved Karl cause I took his place

Reply:
Well well well, starting next sprint I'm in Core Experience apparently, with the goal of making that team a bit more multidisciplinary (and similarly Core Analytics is getting Li if I remember correctly) @timgl